seatunnel-connectors-v2/connector-cdc/connector-cdc-mysql/src/main/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/utils/MySqlTypeUtils.java

```diff
@@ -62,10 +66,17 @@ public static org.apache.seatunnel.api.table.catalog.Column convertToSeaTunnelCo
                         MySqlValueConverters::defaultParsingErrorHandler);
         MySqlDefaultValueConverter mySqlDefaultValueConverter =
                 new MySqlDefaultValueConverter(mySqlValueConverters);
-        Object defaultValue =
-                mySqlDefaultValueConverter
-                        .parseDefaultValue(column, column.defaultValueExpression().orElse(null))
-                        .orElse(null);
+
+        Optional<String> defaultValueExpression = column.defaultValueExpression();
+        Object defaultValue = defaultValueExpression.orElse(null);
```

Comment:
What problem does the modification of this place solve？

Reply:
> What problem does the modification of this place solve？

Support some specialDefaultValue, like CURRENT_TIMESTAMP(3).